src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,15 @@
+$calendar-gap: 1px;
+$border-field: 1px;
+$font-size: 30px;
+$field-width: 100px;
+$space: calc(#{$field-width} + #{$calendar-gap});
+$calendar-width: calc((#{$space} + #{$border-field}) * 7 );
```

Comment:
it would be easier to calculate if you set `box-siizing: border-box` for each element using `*` selector, then you don't need to calculate borders, it would be enough - `$field-width * 7 + $calendar-gap * 6` and variable space would be unnecessary